crates/volta-core/src/tool/node.rs

```diff
@@ -13,11 +15,19 @@ where
     session.add_event_start(ActivityKind::Node);
 
     match session.current_platform()? {
-        Some(ref platform) => {
+        Some(platform) => {
+            match platform.source() {
+                Source::Project => debug!("Using node@{} from project platform", platform.node()),
+                Source::User => debug!("Using node@{} from user default platform", platform.node()),
+            };
+
             let image = platform.checkout(session)?;
             let path = image.path()?;
             Ok(ToolCommand::direct(OsStr::new("node"), args, &path))
         }
-        None => ToolCommand::passthrough(OsStr::new("node"), args, ErrorDetails::NoPlatform),
+        None => {
+            debug!("Could not find platform, delegating to system");
```

Comment:
Probably should include what we are looking for (AFAICT it is `node`, right?).

Reply:
That's a good point, all of the "No platform" messages could be updated to be more specific. Maybe "Could not find volta-managed node, delegating to system"?

Reply:
Ya, that sounds great